src/Icons/doc/index.rst

```diff
@@ -628,6 +628,21 @@ Full Configuration
         # Whether to ignore errors when an icon is not found
         ignore_not_found: false
 
+        # Icon sets configuration, array of prefix => icon set configuration
+        icon_sets:
+            # Icon set for "flags", can be used like this: ux_icon('flags:fr')
+            flags:
+                It can either be mapped to a local directory...
```

Comment:
```suggestion
                # It can either be mapped to a local directory...
```

Reply:
Fixed in https://github.com/symfony/ux/pull/2939, thanks!